BUILD.gn

```diff
@@ -191,7 +152,7 @@ static_library("libdeno") {
   # header file that contains the the sha256 hash of the snapshot.
   if (cc_wrapper != "" && cc_wrapper != "ccache") {
     hash_h = "$target_gen_dir/bundle/hash.h"
```

Comment:
libdeno_tests now also needs this hash dependency, because it now also has a snapshot embedded with an .incbin directive.

Reply:
Fixed. Now test_cc loads the snapshot from disk at runtime instead of using incbin.

Reply:
Because the path to the snapshot is now supplied via pre-processor variable, and because I want to be able to execute test_cc from both the project root and the build root, I've passed the path to snapshot_libdeno_test.bin as an absolute path. This hit your appveyor check, so I whitelisted it.